metis-dereference/metis-dereference-common/src/main/java/eu/europeana/metis/dereference/DereferenceUtils.java

```diff
@@ -26,17 +38,8 @@
 import eu.europeana.corelib.definitions.jibx.SameAs;
 import eu.europeana.corelib.definitions.jibx.TimeSpanType;
 import eu.europeana.corelib.definitions.jibx.WebResourceType;
-import java.util.HashSet;
-import java.util.List;
-import java.util.Set;
-import org.apache.commons.io.IOUtils;
-import org.apache.commons.lang3.StringUtils;
-import org.jibx.runtime.BindingDirectory;
-import org.jibx.runtime.IBindingFactory;
-import org.jibx.runtime.IUnmarshallingContext;
-import org.jibx.runtime.JiBXException;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+import eu.europeana.enrichment.api.external.model.EnrichmentBase;
+import eu.europeana.enrichment.utils.EntityMergeUtils;
 
```

Comment:
Some methods in this class are not covered by unit tests. These methods are fairly straightforward, and typically wrap around another method/library. But maybe there is something we want to check? How to deal with edge cases and what exceptions to expect, and so on?

Reply:
The methods used for the code that will be used in the enrichment topology is covered by unit tests. Further coverage should come after we decide if we're going to refactor/modify things significantly.